spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -43,4 +21,30 @@
 
     expect(rendered).to have_css('.step-3.active')
   end
+
+  context 'when render has no dependencies' do
+    before do
+      render
+    end
+
+    it 'has a localized heading' do
+      expect(rendered).to have_content t('headings.recovery_code')
+    end
+
+    it 'informs the user of importance of keeping the recovery code in a safe place' do
+      expect(rendered).to have_content t('instructions.recovery_code')
+    end
+
+    it 'contains link to continue authenticating' do
+      expect(rendered).
+        to have_xpath("//input[@value='#{t('forms.buttons.continue')}']")
```

Comment:
Oh I see this was just copied from before. Fine to keep as-is. 

If you'd like to change it, I think that a clearer expectation here would be:
`expect(rendered).to have_link(t('forms.buttons.continue'), href: sign_up_recovery_code_path)`

This is testing the same thing as lines 39 -> 42 but is slightly easier to read.